docs/getting_started/installation.rst

```diff
@@ -45,3 +45,9 @@ take advantage of? If so, you can install PRAW directly from GitHub like so:
 .. code-block:: bash
 
    pip install --upgrade https://github.com/praw-dev/praw/archive/master.zip
+
+You can also directly clone a copy of the repository using git, like so:
```

Comment:
I'd add a little more text here that hits the points that (a) you need git installed and (b) this installs a development version.